src/github.com/travis-ci/worker/config.go

```diff
@@ -0,0 +1,65 @@
+package worker
```

Comment:
This file is no longer valid, as it has been reworked and moved into `config/*.go`.  This version can be safely `git rm`'d.